### ISSUE TITLE
Clarification of 'Feature Branch' in PR Guidance

### ISSUE THREAD

**Issue by michael-hawker** [CommunityToolkit/WindowsCommunityToolkit-wiki#40]
@RosarioPulella I think it'd be interesting to specify an example of what a feature branch is for this repo (not the main branch, whichever the name and format, if I understand correctly). "Feature" has very different meanings depending on repositories.

_Originally posted by @jeromelaban in https://github.com/CommunityToolkit/WindowsCommunityToolkit/issues/4148#issuecomment-909424841_

**Comment by michael-hawker** [comment]
For background the reason we have this policy is that we once noticed a whole feature of the Toolkit was deleted between versions once. We noticed later when we browsed the history that someone had deleted these files (for some reason) in their own local main branch.

That wasn't show in the PR difference when comparing their main branch to ours with the new features (for some reason) and therefore the PR was merged with the offending deletes as well which got merged into the history.

Therefore, to ensure a stable environment, we ask that folks branch off of our main into a named non-main branch of their forked repository to start any new feature development. This should better ensure that deletions in a PR review are caught and not merged into our main branch unexpectedly.
